sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 2.9.0-beta.1 (Unreleased)
+## 2.9.0-beta.2 (Unreleased)
```

Comment:
nit: pull in the 2.9.0-beta.1 changelog entry into main, put it where it would have been chronologically

Reply:
thanks for catching that, done!